consensus/src/quorum/task.rs

```diff
@@ -41,9 +41,9 @@ impl Quorum {
     /// Spawn a task to process quorum messages for a specified round
     /// There could be only one instance of this task per a time.
     pub(crate) fn spawn<D: Database + 'static>(
-        &mut self,
+        &self,
         ru: RoundUpdate,
-        provisioners: Provisioners,
+        provisioners: Arc<Provisioners>,
```

Comment:
I disagree, it needs to be Arc since the reference of the variable initialized with `Arc::new(prov.clone()))` needs to be passed across multiple tokio task.

Reply:
I mean you don't need change the signature of the function to achieve this.